website/package.json

```diff
@@ -55,5 +54,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "npx": "^10.2.2",
```

Comment:
I'm not sure why npx is installed. 
I've tried to uninstall it locally and everthing works fine 🤔

Reply:
OK, I will uninstall `npx` since it's pulling a lot of dependencies that we could do without.

Reply:
Yes makes sense. After checking the `npx` repo, it appears that the project has been deprecated as npx is now part of `npm cli`.
https://github.com/npm/npx#readme